page-data.json

```diff
@@ -7,5 +7,6 @@
   "imageHeight": "765",
   "facebookPage": "bankempatii",
   "twitterPage": "BankEmpatii",
-  "twitterCreator": "tomekmagulski"
+  "twitterCreator": "tomekmagulski",
```

Comment:
sugerowałem się tym [przykładem](https://dev.twitter.com/cards/types/summary-large-image) ale może nie jest to wymagane